frontend/static/quotes/french.json

```
@@ -804,6 +804,36 @@
       "source": "René Descartes (1596-1650)",
       "length": 169,
       "id": 133
+    },
+    {
+      "text": "Je n'aurais jamais dû tenter une telle expérience. Les personnages nés de notre imagination portent en eux toutes nos facettes, les plus claires comme les plus sombres. Tous nos désirs de vie, d'amour, de justice... mais aussi d'orgueil, de bêtise, de meurtre. Je ne veux plus me retrouver face à ceux que j'ai créés. L'angoisse de la page blanche est mille fois préférable au miroir que ces êtres nous tendent.",
+      "source": "Quête Impossible de Ann Rocard",
+      "length": 411,
+      "id": 134
+    },
+    {
+      "text": "Et quand je prends ce que l'on appelle le mic', je marque, autant de points qu'il y a de zéros dans les sapes de marque. Animal animé d'âme, de luminosité, c'est sans nervosité que j'ai clamé moins d'animosité.",
+      "source": "MC Solaar - La La La, La",
+      "length": 210,
+      "id": 135
+    },
+    {
+      "text": "J’ai gâché des péloches, Avenue Foch, c’est moche, loupé le coche deux mioches, sur une roche, main dans les poches",
+      "source": "MC Solaar - Zoom",
+      "length": 115,
```

This also needed to be changed :/
```suggestion
      "length": 116,
```